lib/rules/no-unescaped-entities.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @fileoverview HTML special characters should be escaped.
+ * @author Patrick Hayes
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+var DEFAULTS = ['>', '<', '"', '\'', '}'];
```

Comment:
I think `<<div>` is just as problematic as `</div>>`, and `{<Foo />}}` and `{{<Foo />}` so those seem obvious to me (let's add `{`?)

I'm not sure I see the purpose in having double or single quotes in here, though.